sale_comment_template/__manifest__.py

```diff
@@ -6,21 +6,22 @@
 {
     "name": "Sale Comments",
     "summary": "Comments texts templates on Sale documents",
-    "version": "11.0.1.1.0",
-    "depends": [
-        "sale",
-        "account_invoice_comment_template",
-    ],
+    "version": "12.0.1.1.0",
+    "category": "Sale",
     "author": "Camptocamp,"
               "Tecnativa,"
               "Odoo Community Association (OCA)",
     "license": "AGPL-3",
+    "installable": True,
```

Comment:
Why this random reordering of the keys?

Reply:
Randomly?

Ordering is the same as https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

Reply:
And that one was randomly picked... Sorting alphabetically might have sense, but not other one. We can change that on the template for avoiding misunderstandings.

Reply:
What do you mean with randomly picked?

Reply:
What should be changed according to you?

Reply:
On other PR's I get the remark to follow the template as defined in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py.
So this is what I did here...

Reply:
> What do you mean with randomly picked?

There was no express intention in that ordering

> What should be changed according to you?

Leave it as it was for reducing the diff, or sort alphabetically for a more logical one. As said, we can provide that alphabetic order also on the template. Any way, all of this is not needed for the module to work, and that's why I don't usually care about this, except for my diff saving mania :wink: 

> On other PR's I get the remark to follow the template...

I don't think it was my remark, isn't it? If so, consider it a nitpicking, the same as this whole one. If you consider you don't want to do anything more, just tell and I won't say more.

Reply:
Ok, for this one I leave it as it is now!